pkg/parquet/reader.go

```diff
@@ -51,8 +51,6 @@ type s3Reader struct {
 }
 
 func NewReader(ctx context.Context, config cfg.Config, logger log.Logger, settings *ReaderSettings) (*s3Reader, error) {
-	s3Cfg := gosoS3.GetLegacyConfig(config, "default")
```

Comment:
Can we remove the legacy config now? My Goland can't find any further usages

Reply:
good point 

done